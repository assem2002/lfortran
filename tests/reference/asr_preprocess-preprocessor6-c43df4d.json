{
    "basename": "asr_preprocess-preprocessor6-c43df4d",
    "cmd": "lfortran --cpp --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/preprocessor6.f90",
    "infile_hash": "ab74c4b1ba190571a15fc7788dd695d43963753391f8e83f5780477e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr_preprocess-preprocessor6-c43df4d.stdout",
    "stdout_hash": "141cc10a9a101799fa55e2469f533720b7d1749015e3d15e1a75df57",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}