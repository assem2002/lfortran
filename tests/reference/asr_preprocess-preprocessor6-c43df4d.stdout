(TranslationUnit
    (SymbolTable
        1
        {
            preprocessor6:
                (Program
                    (SymbolTable
                        2
                        {
                            
                        })
                    preprocessor6
                    []
                    [(Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "1a"
                                (Character 1 2 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "1b"
                                (Character 1 2 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "2a"
                                (Character 1 2 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "2b"
                                (Character 1 2 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "1b"
                                (Character 1 2 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "10"
                                (Character 1 2 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "1a0"
                                (Character 1 3 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "1ab"
                                (Character 1 3 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "1ac"
                                (Character 1 3 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "1c"
                                (Character 1 2 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "10"
                                (Character 1 2 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "1a0"
                                (Character 1 3 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "1aa"
                                (Character 1 3 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "1ac"
                                (Character 1 3 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "1c"
                                (Character 1 2 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )]
                )
        })
    []
)
